pom.xml

```diff
@@ -573,6 +573,7 @@
                     <groupId>org.asciidoctor</groupId>
                     <artifactId>asciidoctorj</artifactId>
                     <version>1.5.2</version>
+                    <scope>provided</scope>
```

Comment:
With this change, do we still need to explicitly turn off the owasp plugin in the different applications' `docs/pom.xml`? For instance, in `catalog/content/docs/pom.xml`, we have this:

```
            <!-- Disable vulnerability checks since this project only contains documentation -->
            <plugin>
                <groupId>org.owasp</groupId>
                <artifactId>dependency-check-maven</artifactId>
                <executions>
                    <execution>
                        <phase>none</phase>
                        <goals>
                            <goal>check</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
```

Can this be removed?